### DISCUSSION TITLE
Error 400 - POST to upload file in bot framework emulator

### DISCUSSION THREAD

**Discussion by lohanysantt** [microsoft/BotFramework-Composer#9736] [General]
I'm encountering an issue when attempting to upload large files (specifically, files of size 238MB and larger) in the Bot Framework Emulator. While smaller files are successfully uploaded and validated, attempting to upload files larger than approximately 234MB results in a 400 error with the following message:

message: "Cannot read property 'name' of undefined"

code: "ServiceError"

I've thoroughly checked my code, and it functions as expected with smaller files. However, when attempting to upload larger files, the error occurs consistently. Unfortunately, I'm unable to retrieve any error logs as the message doesn't even get sent and remains stuck in the "sending" state.

Could this issue be related to any limitations within the Bot Framework Emulator regarding file size? Or perhaps there's another aspect I might be overlooking? Any insights or suggestions on how to resolve this issue would be greatly appreciated.
<img width="606" alt="Captura de tela 2024-04-22 191344" src="https://github.com/microsoft/BotFramework-Composer/assets/93150042/4652c775-afa1-4336-9b92-451174686d64">

**Comment by OEvgeny** [comment]
Could you try outside of emulator, e.g. by using just webchat? Chances are this is Emulator issue
